tests/testAstropyTableViews.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python2
+from __future__ import absolute_import, division
+
+#
+# LSST Data Management System
+# Copyright 2016 AURA/LSST
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""
+Tests for Astropy views into afw.table Catalogs
```

Comment:
Basically correct.  The tests do not exercise any new Astropy functionality.  We do have a method that will hopefully work with v1.2, but that's untested.

But even once v1.2 is out, I'd like to keep the `asAstropy` (or whatever we call it) method public, as it provides useful customization options that will likely never be available through the Astropy table constructors (since they'd require API changes to those constructors).